packages/firestore/test/util/helpers.ts

```diff
@@ -246,7 +246,14 @@ export function queryData(
   queryPurpose: QueryPurpose,
   path: string
 ): QueryData {
-  return new QueryData(query(path)._query, targetId, queryPurpose);
+  // Arbitrary value.
+  const sequenceNumber = 0;
```

Comment:
Does it make sense to use something other than `0` here for future debugging help?